src/sql/src/session/vars.rs

```diff
@@ -1979,6 +1979,13 @@ feature_flags!(
         internal: true,
         enable_for_item_parsing: true,
     },
+    {
+        name: enable_notices_for_index_already_exists,
+        desc: "emitting notices for IndexAlreadyExists (doesn't affect EXPLAIN)",
+        default: true,
+        internal: true,
+        enable_for_item_parsing: true,
```

Comment:
> I'm wondering whether enable_for_item_parsing should be true. Maybe it doesn't matter, because the flag just affects whether we send notices to the shell (not EXPLAIN or the internal table),

I think it doesn't matter.

> but notices during bootstrapping won't be shown in a shell anyway, right?

Yes, because there are not calling `emit_optimizer_notices()` in the `bootstrap` code.